pylint_django/checkers/forms.py

```diff
@@ -0,0 +1,49 @@
+"""Models."""
+from astroid.nodes import Assign, ClassDef
+
+from pylint.interfaces import IAstroidChecker
+from pylint.checkers.utils import check_messages
+from pylint.checkers import BaseChecker
+
+from pylint_django.__pkginfo__ import BASE_ID
+from pylint_django.utils import node_is_subclass
+
+
+def _get_child_meta(node):
+    for child in node.get_children():
+        if isinstance(child, ClassDef) and child.name == 'Meta':
+            return child
+    return None
```

Comment:
You can remove this line b/c this is the default in Python. Indeed the unreleased pylint 2.0 has a check for this and will complain.

Reply:
Yes, I added it because my installed version of Pylint marked it. I'll take it away.

Reply:
If pylint complains differently in different versions... well now you know why this PR is fixing the hacks I made :-)  see also https://github.com/PyCQA/pylint-plugin-utils/blob/master/pylint_plugin_utils/__init__.py#L109 ...